napari_animation/_qt/keyframeslist_widget.py

```diff
@@ -185,6 +191,14 @@ def _key_frame_to_label(self, key_frame):
         key_frame_id = id(key_frame)
         return self._key_frame_id_to_label_map[key_frame_id]
 
+    @property
+    def _item_background_color(self):
+        return 'QListView::item:selected{background-color: rgb(0,122,204);}'
```

Comment:
Can you get this color from the napari theme - you might need to do something like @tlambert03 did here https://github.com/napari/napari-console/blob/3313a710ad671532727c6d2f4f8d255afdc6bb05/napari_console/qt_console.py#L151-L155.

and connect an update them method here https://github.com/napari/napari-console/blob/3313a710ad671532727c6d2f4f8d255afdc6bb05/napari_console/qt_console.py#L81.

Not how the import is done inside the `_update_theme` method to cheat a little on our dependencies.

I don't all love this and think we might want to find a better way long term to allow plugins to update stylings based on themes, but this is where we are at now :-)